hack/dockerized

```diff
@@ -12,7 +12,8 @@ fi
 
 fail_if_cri_bin_missing
 
-KUBEVIRT_BUILDER_IMAGE="quay.io/kubevirt/builder:2109081533-c2a4c43a6"
+# TODO - change this to remote image
+KUBEVIRT_BUILDER_IMAGE="builder:2112201343-13d494f55"
```

Comment:
Should this be included in the PR?

Reply:
No, it's for local testing only (that's why the TODO is there).

Reply:
I see, it's probably safer to not commit it so it won't accidentally slip in.

Reply:
Ahh, got you, this will be replaced with the proper remote image at a later stage in this PR.

Reply:
Rebuilt the image for you with 1.17.1. You can use `quay.io/kubevirt/builder:2112231314-6355ab2fd`.

Reply:
Thank you @rmohr